CONTRIBUTING.md

```diff
@@ -168,12 +168,14 @@ yarn kbn bootstrap
 
 (You can also run `yarn kbn` to see the other available commands. For more info about this tool, see https://github.com/elastic/kibana/tree/master/packages/kbn-pm.)
 
-Start elasticsearch.
+Start elasticsearch from a nightly snapshot.
 
 ```bash
-yarn elasticsearch
+yarn es snapshot
 ```
 
+Additional options are available, pass `-help` for more information.
```

Comment:
Should be `--help` right?

Reply:
Yup 1762942b7c676eead3c3c9eec5fd8fc43217a287